bindings/gumjs/runtime/console.js

```diff
@@ -15,6 +21,24 @@ class Console {
   error() {
     sendLogMessage('error', slice.call(arguments));
   }
+
+  count(label = 'default') {
+    if (!this.#counters.has(label)) {
+      this.#counters.set(label, 0);
+    }
+
+    const newValue = this.#counters.get(label) + 1;
+    this.#counters.set(label, newValue);
+    this.log(`${label}: ${newValue}`);
+  }
+
+  countReset(label = 'default') {
+    if (!this.#counters.has(label)) {
+      this.warn(`Counter "${label}" doesn't exist.`);
```

Comment:
Perhaps we should reverse these two branches so the happy/normal-case is first.

Also it would be good to `.delete()`, so the `Map` doesn't keep growing in case there's a lot of dynamically generated labels over time.